.gitignore

```diff
@@ -0,0 +1,8 @@
+node_modules
+/log/
+*.log
+.DS_Store
+.eslintrc.js
```

Comment:
Why is the eslint configuration excluded?

Reply:
Fixed